Library/Sender.ts

```diff
@@ -128,59 +137,99 @@ class Sender {
     public saveOnCrash(payload: string) {
         this._storeToDiskSync(payload);
     }
-    
-    private _confirmDirExists(direcotry: string, callback: (err) => void): void {
-        fs.exists(direcotry, (exists) => {
+
+    /**
+     * Send all temp files from last crash.
+     * reads all files that exist, deletes them and tries to send its load
+     */
+    public sendAllFilesOnDisk(): void {
+        var tempDir = Sender.OFFLINE_DIRECTORY || path.join(os.tmpDir(), Sender.TEMPDIR);
```

Comment:
@KamilSzostak, there's a subtlety here on whether _enableOfflineMode flag should apply to saving or whether we should also consider it when reading from disk. I think I'm with you here, that _enableOfflineMode=False means, **don't touch my file system**